### PR TITLE
first issue: add missing comma

### PR DESCRIPTION
**Pull request by gbradleypro**

### CODE REVIEW

File: content/blog/2020-10-20-react-v17.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ author: [gaearon,rachelnabors]
 
 Today, we are releasing React 17! We've written at length about the role of the React 17 release and the changes it contains in [the React 17 RC blog post](/blog/2020/08/10/react-v17-rc.html). This post is a brief summary of it, so if you've already read the RC post, you can skip this one.
 
-## No New Features
+## No New Features {#no-new-features}
```
Comment (eps1lon) [inline content/blog/2020-10-20-react-v17.md:8]: The anchors are working on master as well: https://reactjs.org/blog/2020/10/20/react-v17.html#no-new-features

What problems do explicit markers solve here?

**Review by eps1lon — CHANGES_REQUESTED** [review]
The typo explained in the issue makes sense. But there are some other changes that aren't mentioned. Did you accidentally opened this PR from an outdated branch?

File: content/blog/2020-10-20-react-v17.md (hunk 1 above)
Comment (gbradleypro) [reply content/blog/2020-10-20-react-v17.md:8]: Hey @eps1lon, I noticed this and actually it seems like those changes would just get added on whenever I ran: 
`yarn check-all`

I left it thinking maybe that's part of the linter, but not worries can remove

Comment (eps1lon) [reply content/blog/2020-10-20-react-v17.md]: >  I noticed this and actually it seems like those changes would just get added on whenever I ran:
yarn check-all

Thanks! It's part of `yarn generate-ids`. I included the change while we're at it.

**Review by eps1lon — APPROVED** [review]

File: content/blog/2020-10-20-react-v17.md (hunk 1 above)
Comment (gbradleypro) [reply content/blog/2020-10-20-react-v17.md:8]: Awesome, thank you!

**Comment by eps1lon** [comment]
@gbradleypro Thanks!

**Pull request merged**